src/wasm/CMakeLists.txt

```diff
@@ -5,12 +5,22 @@ faasm_private_lib(wasm
     chaining_util.cpp
 )
 
-# Populate the variables defined for cross-compiling WASM
-set(FAASM_WASM_BYTES_PER_PAGE $ENV{FAASM_WASM_BYTES_PER_PAGE})
-set(FAASM_WASM_CTORS_FUNC_NAME $ENV{FAASM_WASM_CTORS_FUNC_NAME})
-set(FAASM_WASM_MAX_MEMORY $ENV{FAASM_WASM_MAX_MEMORY})
-set(FAASM_WASM_STACK_SIZE $ENV{FAASM_WASM_STACK_SIZE})
-set(FAASM_WASM_ZYGOTE_FUNC_NAME $ENV{FAASM_WASM_ZYGOTE_FUNC_NAME})
+# Shared variables with the cross-compilation toolchain
```

Comment:
@eigenraven is this more or less what you had in mind? Also, is there any cleaner way to check if any is not set?

Reply:
Yes, and the conditions can probably just be `if((NOT FAASM_WASM_BYTES_PER_PAGE) OR (NOT FAASM_WASM_CTORS_FUNC_NAME) )` etc. What you wrote is wrong, because you force expansion with ${}, so it would evaluate to `NOT DEFINED CACHE{65536}` - when passing variables "by reference" to cmake commands you always name them, without quotes or $-expansion.